docs/docs/software-architecture.md

```diff
@@ -149,16 +148,16 @@ Logging:
 
 - To be decided by ChainSquad if it gets outsourced to a dedicated SaaS
 
-
+   
 ## 7. Implementation
 
 The implementation diagram is used to visualize the flow of control and the implementation description elaborates further on the processes taking place.
-
+   
 ### 7.1. Implementation Diagram
 
 ![](img/newImplementation.svg)   
 Figure 5: Implementation Diagram (Swapchain, 2020) [4]
-
+   
 ### 7.2. Implementation Description
 
 End-User:   
```

Comment:
they don't submit an order but rather directly create an HTLC on the respective blockchain when they submit the information filled out in the form